config/samples/mariadb/mariadb-db.yaml

```
@@ -0,0 +1,207 @@
+                name: model-registry-db
+          - name: MYSQL_DATABASE
+            valueFrom:
+              secretKeyRef:
+                key: database-name
+                name: model-registry-db
+          image: quay.io/sclorg/mariadb-105-c9s:latest
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            exec:
+              command:
+                - /bin/bash
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 14 but found 16

(indentation)